Examples/Framework/include/ActsExamples/EventData/Measurement.hpp

```diff
@@ -0,0 +1,57 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/EventData/Measurement.hpp"
+#include "ActsExamples/EventData/IndexSourceLink.hpp"
+
+#include <cassert>
+#include <vector>
+
+namespace ActsExamples {
+
+/// Variable measurement type that can contain all possible combinations.
+using Measurement = ::Acts::FittableMeasurement<IndexSourceLink>;
+/// Container of measurements.
+///
+/// In contrast to the source links, the measurements themself must not be
+/// orderable. The source links stored in the measurements are treated
+/// as opaque here and no ordering is enforced on the stored measurements.
+using MeasurementContainer = std::vector<Measurement>;
+
+/// Calibrator to convert an index source link to a measurement.
+class MeasurementCalibrator {
```

Comment:
I would prefer to leave the `MeasurementCalibrator` name for the following reason: the `IndexSourceLink` is intentionally separate from the `Measurement` and can thus be used to reference different types. Naming it `IndexSourceLinkCalibrator` implies that this is the only way the `IndexSourceLink` can be calibrated. But it is actually the other way round, i.e. the `MeasurementCalibrator` is the only tool that gets you a `Measurement` from a source link. If we decide, e.g. to play around with misalignment/alignment we could have a `MisalignedMeasurementCalibrator` that takes the same source link but produces the measurement in a different way.

To make this distinction clearer, we could rename the calibrator to clarify that it just does the lookup without any actual calibration, e.g. `MeasurementContainerAccessor`. I am not convinced that that will not be even more confusing.

Reply:
Yeah, that makes sense. The `MeasurementContainerAccessor` might be a bit weird to be used for calibration. I‘m not sure if the word 'emulator' might help us for the distinction, say `MeasurementCalibratorEmulator` or `EmulatedMeasurementCalibrator`. But I’m also fine with the current name 'MeasurementCalibrator' since we don't have any other calibrator yet.

Reply:
I'll leave it as-is for now.